frontend/integration-tests/tests/kubevirt/vm.actions.scenario.ts

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-undef */
+
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, isLoaded } from '../../views/crud.view';
+import * as vmActionsView from '../../views/kubevirt/vm.actions.view';
+
+describe('Test VM actions', () => {
+  const leakedResources = new Set<string>();
+  const testVM = {
+    apiVersion: 'kubevirt.io/v1alpha2',
+    kind: 'VirtualMachine',
+    metadata: {
+      name: testName,
```

Comment:
again, can we have a different name than namespace? :) I know it is altered in the tests but this object could be reused in the future.

Reply:
Done.